bears/vcs/git/GitCommitBear.py

```diff
@@ -121,7 +121,8 @@ def check_shortlog(self, shortlog,
             if not match:
                 yield Result(
                     self,
-                    "Shortlog of HEAD commit does not match given regex.")
+                    "Shortlog of HEAD commit does not match given regex.\n" +
+                    shortlog_regex)
```

Comment:
nah this is not well shown, i think it should be somehow in brackets, like:

Shortlog of HEAD commit does not match given regex (regex).